lib/utils/error-messages.js

```diff
@@ -21,7 +21,7 @@ export function getErrorMessage(report, executionPlatformLabel, modeler = 'deskt
   } = report;
 
   if (!error) {
-    return report;
+    return message;
```

Comment:
Is this tested anywhere?

Reply:
I'll add a test. This is implicitly tested whenever a rule has no `error` like the ones provided by plugins.

Reply:
Done (cf. https://github.com/camunda/linting/pull/20/commits/8ec90d205972241218ab51f78657185d54f928c5#diff-2f98e4495e7a632249dece58a33acb12c4c42920203074242e15fa685d1edad8).